refinery/ui/source/js/user-file-browser/services/factory.js

```diff
@@ -22,11 +22,7 @@
       createColumnDefs: createColumnDefs,
       createData: createData,
       createFilters: createFilters,
-      getUserFiles: getUserFiles,
-      _mergeAndAddObject: _mergeAndAddObject,
```

Comment:
remove unused helper methods since I shifted the logic for the attributeFilter. Track the different attributes and keep duplicate fields separate.